dandiapi/api/tests/test_asset.py

```diff
@@ -253,6 +253,35 @@ def test_asset_populate_metadata(draft_asset_factory):
     }
 
 
+@pytest.mark.django_db
+def test_asset_populate_metadata_zarr(draft_asset_factory, zarr_archive):
+    raw_metadata = {
+        'foo': 'bar',
+        'schemaVersion': settings.DANDI_SCHEMA_VERSION,
+    }
+
+    # This should trigger _populate_metadata to inject all the computed metadata fields
+    asset = draft_asset_factory(metadata=raw_metadata, blob=None, zarr=zarr_archive)
+
+    download_url = 'https://api.dandiarchive.org' + reverse(
+        'asset-direct-download',
+        kwargs={'asset_id': str(asset.asset_id)},
+    )
+    s3_url = f'http://localhost:9000/test-dandiapi-dandisets/test-prefix/test-zarr/{zarr_archive.zarr_id}/'  # noqa: E501
```

Comment:
We'll want to use the `DJANGO_MINIO_STORAGE_ENDPOINT` setting here like in https://github.com/dandi/dandi-api/blob/master/dandiapi/api/tests/test_version.py#L94, otherwise this test will fail when run in docker-compose where `DJANGO_MINIO_STORAGE_ENDPOINT` is `http://minio/....`

Reply:
Good point